health-services/facility/README.md

```diff
@@ -1,5 +1,57 @@
-# facility
+# Facility
+
+### Facility Service
+Facility registry is a Health Campaign Service that facilitate warehouse on the DIGIT platform. The functionality is exposed via REST API.
+
+### DB UML Diagram
+
+<img width="541" alt="Screenshot 2023-03-29 at 2 23 24 PM" src="https://user-images.githubusercontent.com/123379163/228481126-e55c4d15-196c-4991-83cf-dffd10d1edc2.png">
+
+### Service Dependencies
+- Idgen Service
+
+### Swagger API Contract
+Link to the swagger API contract yaml and editor link like below
+
+https://editor.swagger.io/?url=https://raw.githubusercontent.com/egovernments/health-campaign-services/v1.0.0/docs/health-api-specs/contracts/registries/facility.yml#!/
+
+### Service Details
+
+#### API Details
+BasePath `/facility/v1`
+
+Facility service APIs - contains create, update, delete and search end point
+
+a) POST `/facility/v1/_create` - Create Facility, This API is internally call from facility controller to create/add a new facility.
+
+b) POST `/facility/v1/bulk/_create` - Create bulk Facility, This API is internally call from facility controller to create/add new facilities in bulk.
```

Comment:
Change this `Create bulk Facility, This API is internally call from facility controller to create/add new facilities in bulk.` to `Create multiple facilities using the provided request data.`